Part3_icon-vis/ex3_0_icon-vis.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8822930b",
+   "metadata": {},
+   "source": [
+    "# Exercise 3.0: icon-vis\n",
+    "prepared by A. Lauber\n",
+    "\n",
+    "In this exercise, we will work with the GitHub repository [icon-vis](https://github.com/C2SM/icon-vis), which was developed by C2SM and MeteoSwiss. It is a collection of easy-to-use Python scripts for plotting data on unstructured grids like the ICON grid. It is based on the library [psyplot](https://psyplot.github.io), which was already introduced in [Exercise 2.2](../Part2_Mapplots/ex2_2_psyplot.ipynb).\n",
+    "\n",
+    "Advantages of plotting on the unstructured grid:\n",
```

Comment:
Just mention is has the same advantages and disadvantages as discussed in ex 2.2?

Reply:
I removed completely as you are right that it is already discussed in ex. 2.2